tools/src/main/scala/scala/scalanative/nir/parser/Base.scala

```diff
@@ -72,8 +72,10 @@ object Base {
   val Int: P[Int]         = neg((HexNum | DecNum).!) map (_.toInt)
   val Long: P[Long]       = neg((HexNum | DecNum).!) map (_.toLong)
   val Infinity: P[String] = P("Infinityf".!) map (_.init)
-  val Float: P[Float]     = neg(Infinity | Literals.Literals.Float.!) map (_.toFloat)
-  val Double: P[Double]   = neg(Infinity | Literals.Literals.Float.!) map (_.toDouble)
+  val Float
+    : P[Float] = neg(Infinity | Literals.Literals.Float.!) map (_.toFloat)
+  val Double
+    : P[Double] = neg(Infinity | Literals.Literals.Float.!) map (_.toDouble)
```

Comment:
@olafurpg That's an unexpected one. I'd expect for type annotation to stay on the same line. i.e.:
```
val Double: P[Double] = 
  neg(Infinity | Literals.Literals.Float.!) map (_.toDouble)
```